ionic-angular-template/src/auth/sign-up/sign-up.component.html

```diff
@@ -43,8 +43,11 @@
                 </ion-label>
               </ion-item>
             </ion-list>
+            <div class="ion-float-right ion-margin-top ion-margin-bottom">Have an account?  
+              <a [routerLink]="['/forgotPassword']">Log in
+              </a>
+            </div>
             <ion-button
-            class="auth-button"
             color="primary"
             expand="block"
             [disabled]="!signUpForm.valid"
```

Comment:
We were discussing to use the `invalid` when we should check for invalid and `valid` when we check for valid, not the `NOT`.